.github/workflows/ci.yaml

```diff
@@ -7,6 +7,13 @@ on:
   pull_request:
 
 jobs:
+  validate:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v2
+    - name: Validate version files
+      run: .github/set-versions.sh -n 0.0.1
```

Comment:
I don't know if I understand what this is doing. `-n` makes it verify, but `0.0.1` isn't the correct version number, so it changes the versions to `0.0.1`, prints some output, then changes it back and passes the test.

Maybe this is just to make sure the single line check passes?

Reply:
It's verifying that it knows how to edit the version files. Making sure that nobody has moved a version file or changed its format. I'll add a comment explaining what's going on here.